src/vs/workbench/parts/files/browser/media/explorerviewlet.css

```diff
@@ -17,6 +17,10 @@
 	overflow: hidden;
 }
 
+.explorer-viewlet .explorer-item-label a, .explorer-viewlet .working-files-item-label {
+	margin-right: 12px;
+}
+
```

Comment:
It’s an additional margin so that the sidebar optimal width isn’t stuck with the end of the largest tree element. I’ll probably move that margin into the code as the CSS margin trick won’t work for the working files optimal width calculation (since we won’t include the path into this calculation).